crates/sui-indexer/src/models/system_state.rs

```diff
@@ -188,9 +187,7 @@ impl TryFrom<DBValidatorSummary> for SuiValidatorSummary {
             next_epoch_primary_address: db.next_epoch_primary_address,
             next_epoch_worker_address: db.next_epoch_worker_address,
             voting_power: db.voting_power as u64,
-            operation_cap_id: ID {
-                bytes: ObjectID::from_str(&db.operation_cap_id)?,
-            },
+            operation_cap_id: ObjectID::from_str(&db.operation_cap_id)?,
```

Comment:
Is this a real Move ObjectID? ooc what are all of the methods with which objectIDs are created? we need domain separators everywhere.

Reply:
This is indexer deserializing data read from the store. So yes it is a real object ID